cli.js

```diff
@@ -158,7 +158,10 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
```

Comment:
I might have actually fixed this line too.

Do we have a test that covers this?

Everything passed on OSX / Node `4.1` without this and without my change (in other words, unless the problem this addresses is specific to an OS / Node version we do NOT have test coverage for it).